swagger.yml

```diff
@@ -1066,6 +1066,9 @@ definitions:
         $ref: '#/definitions/tilgungsplanBausparvertragMitVorausdarlehen'
       maximalerAuszahlungsbetrag:
         $ref: '#/definitions/euro'
+      alternative:
+        description: Pflichtfeld wenn sich das Angebot um ein Downsell handelt, also entweder die Laufzeit verlängert und oder der Kreditbetrag verringert wurde. In diesem Falle ist zusätzlich mindestens eine Anpassungsmeldung zu liefern.
```

Comment:
Anpassung auf die Laufzeit- und Kreditbetragsgrenzen des Produkts sind nicht als Downsell zu markieren. Diese Unterschiedung wird anhand dieser Beschreibung aktuell nicht klar.

Evtl auch Nebensatz hier weglassen und in der Readme/Beispiel genauer erläutern?